frontend/src/form.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+export default function Form() {
+
+// States for registration
+const [name, setName] = useState('');
+const [email, setEmail] = useState('');
+const [password, setPassword] = useState('');
+
+// States for checking the errors
+const [submitted, setSubmitted] = useState(false);
+const [error, setError] = useState(false);
+
+// Handling the name change
+const handleName = (e) => {
+setName(e.target.value);
```

Comment:
What went wrong with indentation in this file? This is very confusing code to read through